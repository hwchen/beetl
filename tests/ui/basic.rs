use beetl::MeltRecord;

#[derive(MeltRecord)]
enum EnumRecord {
    Variant
}

fn main() {}
