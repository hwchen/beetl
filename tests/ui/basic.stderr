error: derive MeltRecord only support structs
 --> $DIR/basic.rs:4:1
  |
4 | / enum EnumRecord {
5 | |     Variant
6 | | }
  | |_^
